Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.10.3
+FROM golang:1.13
 COPY . /go/src/github.com/vinyldns/terraform-provider-vinyldns
 WORKDIR /go/src/github.com/vinyldns/terraform-provider-vinyldns
-CMD ["make", "deps-build"]
+CMD ["make", "build-deps", "package"]
```

Comment:
I believe `CMD ["make", "package"]` will suffice. I don't believe `build-deps` is necessary, as I'm fairly sure `build` (called as a prerequisite to `package`) ensures dependency downloads and installation. Am I mistaken?

Reply:
Retested and you're right.  Addressed in commit.